smos-query/src/Smos/Query/Commands/Agenda.hs

```diff
@@ -113,16 +113,12 @@ insertHourLines now = go [8 .. 18]
 insertNowLine :: ZonedTime -> [AgendaReportLine] -> [AgendaReportLine]
 insertNowLine now = go
   where
-    go = \case
-      [] -> [NowLine]
-      [x] ->
-        if isBefore now x
-          then [NowLine, x]
-          else [x, NowLine]
-      (x : y : zs) ->
-        if isBetween x now y
-          then x : NowLine : y : zs
-          else x : go (y : zs)
+      go = \case
+        [] -> [NowLine]
+        (x : xs) ->
+          if isBefore now x
+            then NowLine : x : xs
+            else x : go xs
```

Comment:
Do I understand correctly that there will be no `NowLine` if the now line should go after the last `AgendaReportLine`?
Is that on purpose?

Reply:
This should not behave like this and the `NowLine` is inserted no matter what.
Indeed, in the recursive `go` function, if no `AgendaReportLine` validate the condition, the empty list stopping condition happens the `NowLine` at the end of `[AgendaReportLine]`.

Reply:
Ah yes, my bad!